.gitlab-ci.yml

```
@@ -0,0 +1,44 @@
+  tags:
+    - dev
+  artifacts:
+    paths:
+      -  builds/
+    expire_in: 1 day
+
+
+push_job:
+  stage: push_to_dockerhub
+  script:
+    - docker login -u $DOCKERHUB_USER -p $DOCKERHUB_PAT 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)